docs/models/model-typing.mdx

```diff
@@ -75,14 +75,6 @@ You only need to use `CreationOptional` & `NonAttribute` on class instance field
 
 Example of a minimal TypeScript project with strict type-checking for attributes:
 
-import CodeBlock from '@theme/CodeBlock';
-import modelInitExample from '!!raw-loader!@site/.sequelize/v7/packages/core/test/types/typescript-docs/model-init.ts';
-
-<CodeBlock className="language-typescript">
```

Comment:
This complete example felt really noisy and redundant now that the rest of the documentation is typescript-first

Reply:
https://github.com/sequelize/sequelize/pull/16194/files